networkx/algorithms/centrality/current_flow_betweenness.py

```diff
@@ -21,16 +21,18 @@
 
 
 @not_implemented_for("directed")
-@py_random_state(7)
+@py_random_state("seed")
 @nx._dispatchable(edge_attrs="weight")
 def approximate_current_flow_betweenness_centrality(
     G,
+    *,
```

Comment:
Unfortunately this breaks backward compatibility - only newly-added kwargs can be key-word only, not the existing ones.

Reply:
The backward-compatible recipe would be to leave the ordering of the existing kwargs, then `*`, then the new kwarg, so:

```python
def approximate_current_flow_betweenness_centrality(
    G,    
    normalized=True,
    weight=None,
    dtype=float,
    solver="full",
    epsilon=0.5,
    kmax=10000,
    seed=None,
    *,
    sample_weight=1,
):
```

Reply:
Done in [f1c72be](https://github.com/networkx/networkx/pull/8007/commits/f1c72be51455183445c27ae78984f7f29943acfd)